client/packages/common/src/ui/forms/JsonForms/components/Select.tsx

```diff
@@ -42,6 +42,7 @@ const UIComponent = (props: ControlProps) => {
           value={data ?? ''}
           onChange={e => handleChange(path, e.target.value)}
           error={!!props.errors}
+          clearable
```

Comment:
Would like to make this dependent on whether or not the element is "required" (i.e. only show on non-required items), but doesn't look like that info is making it down to the local schema/uiSchema.

Reply:
available in `props.config.required` isn't it?

Reply:
Yes that would be great! I think the option to be able to overwrite it would be useful as well. And yes the info is available:

![image](https://user-images.githubusercontent.com/88299195/199851321-c4c86318-bf33-4996-8e57-d21f7f9da2d9.png)